app/src/main/java/rick_and_morty/di/RickAndMortyApiModule.kt

```diff
@@ -40,22 +42,29 @@ private val moshi =
             .addConverterFactory(MoshiConverterFactory.create(moshi))
     }
 
-/*
-    private val moshi =
-        Moshi.Builder()
-        .add(KotlinJsonAdapterFactory())
-        .build()
 
-    private val retrofit: Retrofit by lazy {
-        Retrofit.Builder()
-            .baseUrl(ApiConstants.BASE_URL)
-            .addConverterFactory(MoshiConverterFactory.create(moshi))
-            .build()
+    @Provides
+    fun provideContext(application: Application): Context {
+        return application.applicationContext
     }
 
-    val apiService: CharacterApiService by lazy {
-        retrofit.create(CharacterApiService::class.java)
-    }
 
-*/
+    /*
+        private val moshi =
+            Moshi.Builder()
+            .add(KotlinJsonAdapterFactory())
+            .build()
+
+        private val retrofit: Retrofit by lazy {
+            Retrofit.Builder()
+                .baseUrl(ApiConstants.BASE_URL)
+                .addConverterFactory(MoshiConverterFactory.create(moshi))
+                .build()
+        }
+
+        val apiService: CharacterApiService by lazy {
+            retrofit.create(CharacterApiService::class.java)
+        }
+
+    */
```

Comment:
Commented code is most likely not needed anymore and should be removed.

Reply:
Done